contextbasedrestrictionsv1/context_based_restrictions_v1_examples_test.go

```diff
@@ -0,0 +1,459 @@
+//go:build examples
+// +build examples
+
+/**
+ * (C) Copyright IBM Corp. 2021.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package contextbasedrestrictionsv1_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/platform-services-go-sdk/contextbasedrestrictionsv1"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+//
+// This file provides an example of how to use the Context Based Restrictions service.
+//
+// The following configuration properties are assumed to be defined:
+// CONTEXT_BASED_RESTRICTIONS_URL=<service base url>
+// CONTEXT_BASED_RESTRICTIONS_AUTH_TYPE=iam
+// CONTEXT_BASED_RESTRICTIONS_APIKEY=<IAM apikey>
+// CONTEXT_BASED_RESTRICTIONS_AUTH_URL=<IAM token service base URL - omit this if using the production environment>
```

Comment:
Please add comments for the `TEST_ACCOUNT_ID` and `TEST_SERVICE_NAME` properties which the code is also retrieving.

Reply:
Hi Phil,
thanks for the review!
I added the documentation about the additional config properties.